src/org/zaproxy/zap/extension/ascanrules/resources/Messages.properties

```diff
@@ -29,6 +29,9 @@ ascanrules.testscrosssitescriptv2.otherinfo.nothtml = Raised with LOW confidence
 
 ascanrules.testpersistentxssattack.name=Cross Site Scripting (Persistent)
 ascanrules.testpersistentxssattack.otherinfo=Source URL: {0}
+ascanrules.testpersistentxssattack.json.name = Cross Site Scripting Weakness (Reflected in JSON Response)
```

Comment:
This might need a tweak (same for description), `Persistent in ...`?

Reply:
Thankd